packages/mux-player/src/template.ts

```diff
@@ -134,9 +135,11 @@ export const content = (props: MuxTemplateProps) => html`
       cast-src="${!!props.src ? props.src : props.playbackId ? toMuxVideoURL(props) : false}"
       cast-receiver="${props.castReceiver ?? false}"
       drm-token="${props.tokens?.drm ?? false}"
+      ad-tag-url="${props.adTagUrl ?? false}"
+      allow-ad-blocker="${props.allowAdBlocker ?? false}"
       exportparts="video"
     >
-      ${props.storyboard
+      ${!props.adBreak && props.storyboard
```

Comment:
thought(non-blocking): I get why this is here, but it feels like the wrong place for this to live longer term. We should instead solve this in the theme itself if possible.

Reply:
yes, maybe another location but in the theme is not possible I think. the media is slotted via the forwarding slot so we don't have much control there.